imsim/lsst_image.py

```diff
@@ -86,6 +87,7 @@ def setup(self, config, base, image_num, obj_num, ignore, logger):
         try:
             self.checkpoint = galsim.config.GetInputObj('checkpoint', config, base, 'LSST_Image')
             self.nbatch = params.get('nbatch', 100)
+            self.checkpoint_sampling_factor = params.get('checkpoint_sampling_factor', 10)
```

Comment:
Even moreso here, default should be 1 I think if this option is not given in the config at all.